examples/dkr-create-oracle-xe-server

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+# Variables used for the script
+if [ ! -f `which pwgen` ]; then
+  echo "You must have pwgen on your machine to run this script"
+  exit 1
+fi
+
+CMG_COMMAND=docker
+DKR_IMAGE="gvenzl/oracle-xe"
+DKR_HOST_PORT=$1
+DKR_CNT_NAME=$2
+DKR_DATA_PATH=dbcxe-$DKR_CNT_NAME-dvol
+DKR_ORA_USER_APP=DAAPPUSER
+DKR_ORA_PW=`pwgen -1 16 | tr -d '\n'`
+DKR_ORA_PW_APP=`pwgen -1 16 | tr -d '\n'`
+
+echo "Docker Image ...............:'$DKR_IMAGE'"
+echo "Docker User ................:'$DKR_ORA_USER_APP'"
+echo "Docker Password ............:'$DKR_ORA_PW'"
+echo "Docker Password (app) ......:'$DKR_ORA_PW_APP'"
```

Comment:
If you agree with the above, then this one should probably be changed "Database App User Password", or perhaps just "App User Password" to keep it short.
Either way, the user and the password descriptions should be kept in sync, i.e. print either "Database App User" and "Database App User Password" or just "App User" and "App User Password".